octodns/provider/yaml.py

```diff
@@ -46,6 +47,7 @@ def __init__(self, id, directory, default_ttl=3600, enforce_order=True,
         self.directory = directory
         self.default_ttl = default_ttl
         self.enforce_order = enforce_order
+        self.manage_root_ns = True
```

Comment:
Would prefer that this be passed in as a parameter to `super` rather than setting the parent class's property here.